### DISCUSSION TITLE
Citation Style

### DISCUSSION THREAD

**Discussion by kekoziar** [carpentries/workbench#63] [Q&A]
Are we using any specific citation style for outside content we use?

**Comment by zkamvar** [comment]
I think this is a good question for @karenword, @brownsarahm, and @tobyhodges.

**Comment by tobyhodges** [comment]
I am not aware of a particular style being used, and there doesn't seem to be anything in [the style guide](https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html). I have been making use of footnotes in the lesson development training and trainer training curricula:

```markdown
This is a sentence with a footnote[^1].
This is some more text.

[^1]: This is the text of the footnote.
```

But I cannot claim to have been ardently following any particular style for my references there. 

If anyone has a strong opinion about how this should be done, I'd be very open to it.

**Comment by karenword** [comment]
Thanks for the tag @zkamvar - we are interested in developing citation processes (eventually) but don't have a chosen style. I wonder if @OscarSiba or @acrall might have an opinion as this is also something that could be relevant to [The Carpentries Style Guide](https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#)

**Reply by acrall** [reply]
I will chat with @OscarSiba about it and get back to everyone

**Reply by acrall** [reply]
@oscarSiba and I are meeting about this and had a question: What type of material is generally cited (e.g. code, publication, dataset)?

**Reply by kekoziar** [reply]
All of those, but also images. Some images that supplement the lesson story are community created, some are not and pull from intellectual property of others.
